orange/src/main/java/com/hartwig/hmftools/orange/algo/virus/VirusInterpreter.java

```diff
@@ -0,0 +1,17 @@
+package com.hartwig.hmftools.orange.algo.virus;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+import com.hartwig.hmftools.common.virus.AnnotatedVirus;
+
+import org.jetbrains.annotations.NotNull;
+
+public class VirusInterpreter
```

Comment:
A class with only static methods should be declared `final` to avoid inheritance